compose/cli/main.py

```diff
@@ -263,14 +263,17 @@ def build(self, options):
         Usage: build [options] [--build-arg key=val...] [SERVICE...]
 
         Options:
+            --build-arg key=val     Set build-time variables for services.
             --compress              Compress the build context using gzip.
             --force-rm              Always remove intermediate containers.
+            -m, --memory MEM        Sets memory limit for the build container.
```

Comment:
Heterogeneous Set/Sets

Reply:
Actually, this is a pre-existing `typo`, but I'll remove the 's'